src/UglyToad.PdfPig.Tokenization/Scanner/CoreTokenScanner.cs

```diff
@@ -287,7 +287,7 @@ public IReadOnlyList<byte> RecoverFromIncorrectEndImage(long lastEndImageOffset)
             return data;
         }
 
-        private IReadOnlyList<byte> ReadInlineImageData()
+        private List<byte> ReadInlineImageData()
```

Comment:
why use `List<byte>`? Is it necessary?

Reply:
This eliminates possible virtual call and wasn't on the public API, but agree it breaks consistency with the rest of the codebase. Let me know if we should revert this one.